sdk/keyvault/azure-keyvault-certificates/samples/issuers_async.py

```diff
@@ -95,6 +95,8 @@
 
     finally:
         print("\nrun_sample done")
+        await credential.close()
+        await client.close()
```

Comment:
Previously before this addition, things were not closed until process termination? Why not demonstrate the new context manager usage in some/all of these?

Reply:
Correct, nothing was closed before termination. I haven't demonstrated context manager usage here because I'm trying to make the smallest change necessary to silence aiohttp warnings. I expect a future overhaul of the samples will include demonstrating clients as context managers.